.changeset/nasty-elephants-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Delete image original that are not used outside of the optimization pipeline
```

Comment:
```suggestion
Updates the Image Services API to now delete original images from the final build that are not used outside of the optimization pipeline. For users with a large number of these images (e.g. thumbnails), this should reduce storage consumption and deployment times.
```
Is this correct?  😅 Just adding a bit more here because this is a minor and Matthew might want a quick line for the blog post content.

Reply:
It is correct! The only nit is that it's not the image service API that does it, but it's around there anyway, so, it's not too bad